samcli/lib/hook/utils.py

```diff
@@ -0,0 +1,28 @@
+"""
+Hook-related utils
+"""
+
+from typing import Dict, Optional, Any
+
+HOOK_METADATA_KEY = "AWS::SAM::Hook"
+
+
+def get_hook_metadata(template: Dict) -> Optional[Any]:
```

Comment:
the type hint of the return was supposed to be `Optional[Dict]` but mypy complained because technically it's `Any` since we're just getting the value of a dict at some key.

Reply:
I think if you defined the `template` parameter as `Dict[str, Dict]` will solve it

Reply:
that worked, thank you!